pkg/workloads/onnx_serve/api.py

```diff
@@ -162,18 +166,22 @@ def predict(app_name, api_name):
                 else:
                     result.append(model_output)
 
-            if request_handler is not None and util.has_function(request_handler, "postinference"):
-                result = request_handler.postinference(result, output_metadata)
+            if request_handler is not None and util.has_function(request_handler, "post_inference"):
+                result = request_handler.post_inference(result, output_metadata)
```

Comment:
Note: log the input to the post processing function (if it's defined) (also in `tf/api.py`)